test/e2e/os/testdata/nodeadm-wrapper.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+AWS_ENDPOINT_URL_EKS={{ .EKSEndpoint }} /tmp/nodeadm-bin "$@"
```

Comment:
can we avoid having two scripts by adding some condition in `test/e2e/os/testdata/nodeadm-init.sh` so if the endpoint variable is set then it's passed to nodeadm but if not, then it isn't?